### DISCUSSION TITLE
[PSD10] Exercise 2.41

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#167] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210060810-3658911c-b8fe-48c0-9ffd-f5609031ed16.png)

**Comment by Elia-Belli** [comment]
$Y = \bar A \bar B \bar C + ABC$
![MUX](https://user-images.githubusercontent.com/107944476/211665952-1fccd340-daf7-4cc7-93a0-94e175ede95a.jpg)

**Comment by fraxb04** [comment]
scusami @Elia-Belli  potresti spiegarmi il punto B, dato che non riesco a capire la logica dietro? 
il B come lo hai implementato?

**Comment by Elia-Belli** [comment]
Mi sono accorto ora che al posto di $BC$ come selettori ho inserito $CD$, il funzionamento rimane comunque invariato.
@fraxb04 utilizzando $BC$ come selettori ho 4 combinazioni possibili e vado a vedere che valori assume in quella combinazione, nella tabella sarà presente 2 volte, una per $A=0$ e una per $A=1$, se i 2 casi hanno stesso output metto $0$ o $1$, altrimenti saranno sicuramente opposti: ad esempio per $BC=00$ la funzione vale $1$ se $A=0$ e vale $0$ se $A=1$ quindi seleziono il valore di $\bar A$ che sarà sempre l'output giusto.

**Comment by fraxb04** [comment]
@Elia-Belli ok, grz corretto
